clients/drcachesim/common/options.cpp

```diff
@@ -358,6 +358,12 @@ droption_t<bytesize_t> op_retrace_every_instrs(
     "TRACE_MARKER_TYPE_WINDOW_ID markers.  For -offline traces, each window is placed "
     "into its own separate set of output files, unless -no_split_windows is set.");
 
+droption_t<std::string> op_trace_instr_intervals_file(
+    DROPTION_SCOPE_CLIENT, "trace_instr_intervals_file", "",
+    "File containing instruction intervals to trace.",
+    "File containing instruction intervals to trace in csv format.  "
+    "Intervals are specified as <start, duration> pairs.");
```

Comment:
with a newline I assume.

Maybe point to clients/drcachesim/tests/instr_intervals_example.csv as an example.

Reply:
Done.